eng/docs/api/assets/DoxygenLayout.xml

```diff
@@ -0,0 +1,226 @@
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.8.17 -->
```

Comment:
I thought we were on 1.8.18?

Reply:
We are. I had the old version installed locally for reasons I cannot remember. I'll re-generate this layout file.